README.md

```diff
@@ -1075,3 +1075,217 @@ Response:
+                "setHostPassword": false,
+                "ipType": "DHCP",
+                "ipAddress": "",
+                "subnetMask": "",
+                "defaultGateway": ""
+            },
+            "createdAt": "2023-11-09T10:32:30.441178Z",
+            "updatedAt": "2023-11-09T10:32:36.241597Z",
+            "refreshedAt": "2023-11-09T10:41:23.770379Z",
+            "type": "virtual-continuous-host-protector",
+            "resourceUri": "/disaster-recovery/v1beta1/virtual-continuous-host-protectors/d7cf09d2-16db-35e3-b552-e43dcfe4d974",
+            "generation": 3
+        }
+    ],
+    "limit": 20000,
+    "offset": 0,
+    "total": 1
+}
+```
+
+#### GET Cluster Data
+GET /disaster-recovery/v1beta1/virtual-continuous-cluster-protectors/
+
+Response:
+```
+{
+  [
+    "id":"string",
+    "name":"string",
+    "associatedVma": {
+            "id":"UUID",
+            "name": "string",
+            "address": "string"
+            },
+    "defaultConfiguration": {
+        "network": "uuid",
+        "datastore": "uuid",
+        "setHostPassword": "bool",
+        "ipType": "string", Statsic or DHCP
+        "ipRangeStart": "string",
+        "ipRangeEnd": "string",
+        "subnetMask": "string"
+        "defaultGateway": "string"
+      },
+  ]
+}
+```
+
+#### Deploy VRA (Host)
+POST /disaster-recovery/v1beta1/virtual-continuous-host-protectors/{hostID}/deploy
+
+Body: 
+```
+{
+"network": "c65d935a-9a6c-335e-a01a-48f55012701e",
+"datastore": "1a789dda-3b60-307d-8f77-e3c3d9ab28cd",
+"setHostPassword": true,
+"hostRootPassword": "QAD@ta2023!#", --> not mandatory and depand on the set above 
+"ipType": "Static",
+"ipAddress": "10.171.11.27",
+"subnetMask": "255.255.240.0",
+"defaultGateway": "10.171.15.254"
+}
+```
+Response: 
+```
+{
+    "taskUri": "/api/v1/tasks/27126c1a-4fc9-4f6b-9a67-373567f70e73"
+}
+```
+
+#### Deploy VRA (Cluster)
+POST /disaster-recovery/v1beta1/virtual-continuous-cluster-protectors/{clusterID}/deploy
+
+Body:
+```
+{
+    "network": "uuid",
+    "datastore": "uuid",
+    "setHostPassword": "bool",
+    "hostRootPassword": "string", 
+    "ipType": "string",--> Statsic or DHCP
+    "ipRangeStart": "string",
+    "ipRangeEnd": "string",
+    "subnetMask": "string",
+    "defaultGateway": "string",
+    "setAsDefaultConfiguration": "bool"
+}
+```
+Response: 
+```
+{
+    "taskUri": "/api/v1/tasks/1d58c8ab-a318-4591-9b48-b8909c2dff7d"
+}
+```
+
+#### Edit VRA (Host)
+PUT /disaster-recovery/v1beta1/virtual-continuous-host-protectors/{hostID}
+
+Body: 
+```
+{
+    "setHostPassword": "bool",
+    "hostRootPassword": "string",
+    "ipType": "string",--> Statsic or DHCP
+    "ipRangeStart": "string",
+    "ipRangeEnd": "string",
+    "subnetMask": "string"
+    "defaultGateway": "string"
+}
+```
+
+Response: 
+```
+{
+    "taskUri": "/api/v1/tasks/e63b3f81-5ada-40b5-82ad-420af30715ed"
+}
+```
+
+#### Edit VRA (cluster)
+PUT /disaster-recovery/v1beta1/virtual-continuous-cluster-protectors/{clusterID}
```

Comment:
We don't currently have Edit Cluster, and I don't know when we'll do
Anyway, there's the typo in Statsic

Reply:
Lior added this API to the story.